.travis.yml

```diff
@@ -9,33 +9,32 @@ branches:
 before_install:
 - pip install --upgrade pip setuptools wheel
 stages:
```

Comment:
Wat's the reason for the changes in these file?

Reply:
i modified the travis.yml because the pylint and mypy tests were failing and therefore the builds were not tested. so i created a travis-ci instance to include the build tests anyway.
i did not kniow that this commit was also included in the pull request.